src/Uno.UWP/ApplicationModel/Background/BackgroundTaskInstance.Android.cs

```diff
@@ -0,0 +1,47 @@
+﻿
+#if __ANDROID__
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Windows.ApplicationModel.Background
+{
+	public partial class BackgroundTaskInstance : IBackgroundTaskInstance
+	{
+		internal BackgroundTaskRegistration Task { get; set; }
+
+		BackgroundTaskRegistration IBackgroundTaskInstance.Task => Task;
+
+		Guid IBackgroundTaskInstance.InstanceId => throw new NotImplementedException();
+
+		uint IBackgroundTaskInstance.Progress { get => throw new NotImplementedException(); set => throw new NotImplementedException(); }

```

Comment:
Keep the notimplemented attribute, same below.

Reply:
notimplemented is not removed from Generated\IBackgroundTaskInstance.cs, but VStudio requires implementation of InstanceId and Progress from IBackgroundTaskInstance.